tests/unit/orders-update.spec.js

```diff
@@ -1,12 +1,12 @@
 import OrdersUpdate from 'index'
+import sinon from 'sinon'
 import { SphereClient } from 'sphere-node-sdk'
 import test from 'tape'
 
-let PROJECT_KEY
-if (process.env.CI === 'true')
-  PROJECT_KEY = process.env.CM_PROJECT_KEY
-else
-  PROJECT_KEY = process.env.npm_config_projectkey
+import orderSample from '../helpers/order-sample.json'
+
+const PROJECT_KEY =
```

Comment:
Getting rid of sphere branding but I think `COMMERCETOOLS_PROJECT_KEY` is a tad long.